### DISCUSSION TITLE
APM

### DISCUSSION THREAD

**Discussion by eroltoker** [brefphp/bref#1421] [General]
Is there a good way to do APM (application performance monitoring) that is compatible with this framework and lambda?

**Comment by t-richard** [comment]
Third party APM support is provided through extra extensions layer. 

Bref supports New Relic and Blackfire at the moment.

See https://github.com/brefphp/extra-php-extensions

**Comment by mnapoli** [comment]
To complement @t-richard's answer, I am also working on docs for monitoring: https://github.com/brefphp/bref/pull/1422 And I am working on adding support for Tideways (PHP APM), which should hopefully land in the next few days/weeks.

**Reply by mnapoli** [reply]
@eroltoker you can now use Tideways with Bref, and there's a new documentation page about monitoring: https://bref.sh/docs/monitoring.html
